Library/Homebrew/formula.rb

```diff
@@ -2918,8 +2918,34 @@ def service(&block)
     #
     # If `satisfy` returns `false` then a bottle will not be used and instead
     # the {Formula} will be built from source and `reason` will be printed.
-    def pour_bottle?(&block)
+    #
+    # Alternatively, a preset reason can be passed as a symbol:
+    # <pre>pour_bottle? :default_prefix_required</pre>
+    # <pre>pour_bottle? :clt_required</pre>
+    def pour_bottle?(requirement = nil, &block)
```

Comment:
```suggestion
    def pour_bottle?(reason: nil, &block)
```

maybe? We use that elsewhere and it feels appropriate here.